tenzies/src/App.js

```diff
@@ -0,0 +1,74 @@
+import React from "react"
+import Die from "./components/Die"
+import {nanoid} from "nanoid"
+import Confetti from 'react-confetti'
+
+export default function App() {
+    const [dice, setDice] = React.useState(allNewDice())
+    const [tenzies, setTenzies] = React.useState(false)
+    
+    React.useEffect(() => {
+        const allHeld = dice.every(die => die.isHeld)
+        const allSameValue = dice.every(die => die.value === dice[0].value)
+
+        if (allHeld && allSameValue) {
+            setTenzies(true)
+        }
+
+    }, [dice])
+
+    // this is a helper function to avoid having repetitive code
+    function generateNewDie() {
```

Comment:
Who is dying?
I think it should be` generateNewDice`? (ending with `dice`, not `die`).

Reply:
should be "die", die is singular